languages/go/exercises/concept/time-duration/.meta/example.go

```diff
@@ -0,0 +1,42 @@
+package _meta
+
+import (
+	"fmt"
+	"time"
+)
+
+// StartTime parses a textual representation of a time duration of the class into the corresponding time.Duration format and return the duration:
+func StartTime(date string) time.Duration {
+	complex, _ := time.ParseDuration("1h10m10s")
+	return complex
+}
+
+// IsOver function that takes the start time and a given time and returns out whether the current class is over or not 
+func IsOver(date string,date1 string) bool {
```

Comment:
The names of these parameters are fairly generic. Could you perhaps come up with more description names? This also applies to the other function parameters.